recipes/diego/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+chmod +x DIEGO/*.py
+chmod +x DIEGO/*.pl
+
+mv DIEGO/*.py ${PREFIX}/bin/
+mv DIEGO/*.pl ${PREFIX}/bin/
+
+touch LICENSE					# This has to be removed once the LICENSE file was added upstream, so far this information is only available at the website
```

Comment:
Hi, could you please add the LICENSE either in the upstream repository or the full text of the GNU GPL v2.0?

Reply:
Waiting for upstream